src/main/java/com/mojang/brigadier/builder/ArgumentBuilder.java

```diff
@@ -53,12 +62,22 @@ public T executes(final Command<S> command) {
     }
 
     public T requires(final Predicate<S> requirement) {
-        this.requirement = requirement;
+        requirements.add(requirement);
         return getThis();
     }
 
+    public T requiresAll(final Iterable<Predicate<S>> requirements) {
+        requirements.forEach(this.requirements::add);
+        return getThis();
+    }
+
+    @Deprecated
     public Predicate<S> getRequirement() {
-        return requirement;
+        return requirements.stream().reduce(__ -> true, Predicate::and);
```

Comment:
If you have added the static method `alwaysTrue()`, you can consider calling it here.